mapic/utils.go

```diff
@@ -0,0 +1,58 @@
+package mistapiconnector
+
+import (
+	"github.com/livepeer/go-api-client"
+	"sync"
+	"time"
+)
+
+const lapiCacheDuration = 1 * time.Second
```

Comment:
Do we need the latest version of a stream for any config that can be changed? Maybe this cache could be even higher

Reply:
Not sure. I'm just afraid to set it higher (and 1s is enough for what we encountered)